themes/bootstrap3/templates/search/pagination.phtml

```diff
@@ -1,14 +1,14 @@
 <?php if ($this->pageCount): ?>
   <nav aria-label="<?=$this->transEscAttr('pagination_label')?>">
-    <ul class="pagination">
+    <ul class="pagination<?=$this->results->getOptions()->loadResultsWithJsEnabled() ? ' js-pagination' : ''?>">
       <?php if (isset($this->previous)): ?>
         <?php if (!isset($this->options['disableFirst']) || !$this->options['disableFirst']): ?>
           <li role="none">
-            <a href="<?=$this->currentPath() . $this->results->getUrlQuery()->setPage(1)?>" aria-label="<?=$this->transEscAttr('page_first')?>">[1]</a>
+            <a href="<?=$this->results->getUrlQuery()->setPage(1)?>" aria-label="<?=$this->transEscAttr('page_first')?>">[1]</a>
```

Comment:
Is it possible there was a reason for using currentPath() here? Perhaps it was always unnecessary, but I have a gut feeling that maybe there's some edge case where it matters.

Reply:
I can't think of any. We used to have the full path in quite a few places, but in Finna we got rid of them years ago without issues. Doesn't mean there can't be reason, I just can't imagine why it would be needed. The only case would be if we used the `base` tag, but fortunately we don't.

Reply:
Thanks, @EreMaijala, it's probably okay then. And interesting about the `base` tag -- I somehow had never encountered that before!

Reply:
@demiankatz That's for the better. I think the best use for the tag is to sow confusion. :)